packages/dds/tree/src/feature-libraries/index.ts

```diff
@@ -117,7 +117,7 @@ export {
 	genericFieldKind,
 	type HasFieldChanges,
 	type NodeExistsConstraint,
-	NodeExistenceState,
+	NodeAttachState,
```

Comment:
Doesn't seem like this export is needed.

Reply:
Removed